trlc/parser.py

```diff
@@ -1835,16 +1835,6 @@ def parse_preamble(self, kind):
             self.stab.register(self.mh, pkg)
         else:
             pkg = self.stab.lookup(self.mh, self.ct, ast.Package)
-            if pkg.declared_late and kind == "trlc":
-                # lobster-trace: LRM.Duplicate_Late_Packages
```

Comment:
this is a trace to the LRM
please have a look at the LRM and also remove the linked content in the LRM